docs/development/release.md

```diff
@@ -6,13 +6,59 @@ We use a global versioning approach, meaning version numbers in cumulus are cons
 
 Read more about the semantic versioning [here](https://docs.npmjs.com/getting-started/semantic-versioning).
 
+## Pre-release testing
+
+Before releasing a new major version of Cumulus that contains breaking changes, we should test the deployment upgrade from the latest stable release of Cumulus to the upcoming release with breaking changes.
```

Comment:
Wasn't our intention here to be PI-stable -> PI stable, not all major versions?

Reply:
I guess by "stable" I didn't mean PI stable, though I can see the confusion. I was just saying that if you're releasing a new major major version (6.0.0), we need to at least be sure that the upgrade path from the latest release (5.0.1) is successful. For each PI, we may also need to guarantee that going from last PI release to new PI release is successful, but that's not what I was trying to discuss here.

Would this help (just removing the word "stable" to avoid confusion)?

```suggestion
Before releasing a new major version of Cumulus that contains breaking changes, we should test the deployment upgrade from the latest release of Cumulus to the upcoming release with breaking changes.
```

Reply:
I think that updated wording makes the intent more clear - I'm not (yet) convinced that in all cases this test makes sense, particularly given a per-sprint cadence but that's a discussion we can have offline (arch meeting).

Reply:
35d6eb91364027eb7e5441a4823817672a3d8223